public/datasets/billionaires.json

```diff
@@ -91,18 +91,19 @@
       "description": "Whether their wealth was inherited and where from, e.g. 'not inherited', 'father', '3rd generation'"
     }
   ],
+  "defaultLabelColumn": "Wealth Inherited",
   "card": {
     "description": "Multi-decade data on individual billionaires, ranked for each year, with features like age, industry, family wealth, and wealth type (e.g. company founder in the finance industry).",
     "source": "CORGIS Dataset Project https://corgis-edu.github.io/corgis/csv/billionaires/()",
-    "last_updated": "2016",
+    "lastUpdated": "2016",
     "context": {
-      "created_by": "Ryan Whitcomb",
-      "preparation": "Yes",
-      "confidentiality": "Yes",
-      "subgroup_identifiers": "Yes",
-	    "representative": "No",
-      "potential_uses": "Predict the type of a billionare's wealth based on other features.",
-      "potential_misuses": "For example, determining whether men are 'better' at becoming billionaires than women. Women are underrepresented among billionaires, but there are a lot of reasons for that the data doesn't reflect."
+      "createdBy": "Ryan Whitcomb",
+      "preparation": true,
+      "confidentiality": true,
+      "subgroupIdentifiers": true,
+	    "representative": false,
```

Comment:
nit: is the indentation off here?

Reply:
same comment for other datasets

Reply:
Huh. This is weird - it shows up okay for me, but I'm using a windows system and I know that sometimes adds little hidden spaces that I don't always see...

Maybe before everything goes live, we can pretty-print the JSON to make sure it's good. I don't think if effects how the program runs though, right?